helm/ffc-demo-payment-web/templates/config-map.yaml

```diff
@@ -0,0 +1,29 @@
+{{- include "ffc-helm-library.config-map" (list . "ffc-demo-payment-web.config-map") -}}
```

Comment:
"ffc-helm-library.config-map" change to ""ffc-helm-library.containter-config-map""